gtf-ignored-keys.json

```diff
@@ -3,5 +3,7 @@
   "eoy-name-7", "eoy-quote-8", "eoy-name-8", "advanced", "beginner", "intermediate", "Webpages with Thimble", "Are you sure you want to delete this make?",
   "Confirm", "Created by", "created", "Updated", "updated", "unpublished", "Resources", "Resource Templates",
   "Remixes with X-Ray Goggles", "NoEventsFound", "Multimedia with Popcorn", "make", "Make", "Details", "Explore",
-  "DetailsShareTwitterMsg", "guide", "Info"
+  "DetailsShareTwitterMsg", "guide", "Info", "Did you know Webmaker is also available in", "Webmaker", "StandardFooterTranslate",
+  "StandardConnectingCollaboratingPoint1", "StandardConnectingCollaboratingPoint2", "StandardConnectingCollaboratingPoint3",
+  "StandardFooterButton", "StandardMainDesc1", "StandardMainDesc2", "StandardMainDesc3", "StandardMainHeader"
```

Comment:
This is a pretty confusing situation. We need to mention things like `StandardMainDesc1` here, even though it's translated by an angular filter? Can we just prevent `public/views/literacy.html` from being checked or something?